sdk/logical/response_util.go

```
@@ -203,7 +203,7 @@ func RespondErrorAndData(w http.ResponseWriter, status int, data interface{}, er
 
 	type ErrorAndDataResponse struct {
 		Errors []string    `json:"errors"`
-		Data   interface{} `json:"data""`
```

Was this a problem? Or did Go silently ignore it?

---

It was silently ignored: https://go.dev/play/p/aCrsHnHtj18